src/components/User/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as authenticationClient from '../../../clients/user-client/authentication';
+
+import Popup from '../../UI/Popup/Popup';
+import Button from '../../UI/Button/Button';
+import styleClasses from './LoginForm.module.css';
+import AuthContext from '../../../store/auth-context';
+import Input from '../../UI/Input/Input';
+
+LoginForm.propTypes = {
+    isVisible: PropTypes.bool,
+    onClose: PropTypes.func
+};
+
+function LoginForm(props) {
+    const context = React.useContext(AuthContext);
+
+    const initialState = {
```

Comment:
it's better to move deafault values outside component if they don't depend on it